envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -116,4 +116,10 @@ class AuthorizationProperties {
 
 class EgressProperties {
     var clusterNotFoundStatusCode = 503
+    var automaticHttp2BetweenEnvoysProperties = AutomaticHttp2BetweenEnvoysProperties()
+}
+
+class AutomaticHttp2BetweenEnvoysProperties {
```

Comment:
I don't understand `AutomaticHttp2BetweenEnvoysProperties` maybe there should be some `Http2Properties`. You should also add some description about this feature because it might be unclear to others.

Reply:
Suggestions are welcomed :D I'm not good at naming things.

Reply:
The description is in the docs